src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,66 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+    // ---요구사항1---
+    // size() 메소드를 이용해 Set의 크기 확인하기
+    @Test
+    @DisplayName("요구사항1_size()메소드 이용하기")
+    void size(){
+        int result = numbers.size();
+        assertThat(result).isEqualTo(3);
+    }
+
+    // ---요구사항2---
+    // contains()를 이용한 값 확인 및 ParameterizedTest를 활용한 중복제거
+    @Test
+    @DisplayName("요구사항2_contains()를 이용한 값 확인")
+    void contains(){
+        assertThat(numbers.contains(1)).isTrue();
+        assertThat(numbers.contains(2)).isTrue();
+        assertThat(numbers.contains(3)).isTrue();
```

Comment:
위 피드백대로 import문을 변경하면 아래처럼 간단히 작성해볼 수 있습니다!
```suggestion
        assertThat(numbers).contains(1, 2, 3);
```